README.md

```diff
@@ -84,8 +84,8 @@ If you'd like to chat, we have a [community slack](http://amserializers.herokuap
 Thanks!
 
 ## Documentation
-- [0.10 (master) Documentation](https://github.com/rails-api/active_model_serializers/tree/master)
```

Comment:
Gooooooooood point. Would you like me to create a link for `[0.10.3]` instead and keep master there? Or shall I just rename it?